README.md

```diff
@@ -1,12 +1,8 @@
 # GOV.UK country and territory autocomplete - what it is and how to use it
 
----
-
-**This component is not currently maintained and we are unable to respond to issues and pull requests.**
+The country and territory autocomplete is a component that helps users choose countries and territories in answers. You should only use this component for services where users have to provide their country or territory.
```

Comment:
```suggestion
The country and territory autocomplete is a component that helps users choose countries and territories in answers. You should only use this component for services where users have to provide their country or territory.

It includes complete functionality to make it faster and easier for users to find a location. For example, to select their country of birth or their current location.
 ```
 
 ... and then delete paragraph by line 7.

Reply:
That sounds like a good idea, thanks - will add in.